File: rust/src/src/base/field_permit/field_permit_resolver.rs
use color_eyre::eyre::Result;

use super::field_permit_service::get_field_permit as get_field_permit_service;

/// 字段权限
pub async fn get_field_permit(
  route_path: String,
) -> Result<Option<Vec<String>>> {
  
  let fields = get_field_permit_service(route_path).await?;
  
  Ok(fields)
}

File: deno/src/base/field_permit/field_permit.resolver.ts
import {
  getFieldPermit as getFieldPermitService,
} from "./field_permit.service.ts";

// 字段权限
export async function getFieldPermit(
  route_path: string,
): Promise<string[] | null> {
  
  const fields = await getFieldPermitService(route_path);
  
  return fields;
}
